src/worker/core/phase/newPhaseBeforeDraft.ts

```diff
@@ -50,6 +50,46 @@ const newPhaseBeforeDraft = async (
 			await idb.cache.players.put(p);
 		}
 	}
+	const challengeLoseBestPlayer = g.get("challengeLoseBestPlayer");
+	if (challengeLoseBestPlayer) {
+		const tids = g.get("userTids");
+		for (let i = 0; i < tids.length; i++) {
+			const tid = tids[i];
```

Comment:
`for (const tid of tids) {` is more concise and clear, unless you need the `i` counter for some reason.